src/basictypes.h

```diff
@@ -45,6 +45,9 @@ typedef uint32_t			angle_t;
 #define GCCPRINTF(a,b)
 #define GCCFORMAT(a)
 #define GCCNOWARN
+#ifndef PRId64
```

Comment:
What is with the `#ifndef` guard here, when it is nowhere else in the file? I believe the file itself already has a guard, this seems a little excessive and redundant.

(And on that note, perhaps later on, outside of this pull request we should guard the file with a `#pragma once` rather than its current existing guard)

Reply:
I tried w/o first but noticed a warning on the related travis (win) build for already defined constant.